_posts/2017-11-16-cockpit-156.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Cockpit 156
+author: pitti
+date: 2017-11-16 08:00
+tags: cockpit linux
+slug: cockpit-156
+category: release
+summary: Cockpit with navigation redesign
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](http://cockpit-project.org/). We release regularly.
+Here are the release notes from version 156.
+
+### Redesign navigation and support mobile browsing
+
+The top-level dashboard menus and second-level menus of them have been
+redesigned to clarify the structure, be more extensible, and use less vertical
+screen space:
+
+![desktop mode](/images/mobilenav-desktop.png)
+
+This version introduces proper support for using on mobile phones and tablets.
+As part of this, the login screen was rearranged in mobile mode to avoid
+overlapping the on-screen keyboard with the user/password input fields:
+
+| ![mobile mode with menu](/images/mobilenav-mobile-menu.png) | &nbsp; | ![mobile login page](/images/mobilenav-login.png) |
+
+### Add project homepage link to Apps page
+
+Applications on the Apps page now show a link to the project home page, if
+provided in the AppStream metadata:
+
+![apps homepage link](/images/apps-homepage-link.png)
+
+Thanks Benjamin Deering for this feature!
+
+### Support alternate Kerberos keytabs
+
+If present, Cockpit will now use `/etc/cockpit/krb5.keytab` instead of the
+default system keytab `/etc/krb5.keytab`. This allows users to symlink
+alternate keytabs such as `/etc/httpd/conf/ipa.keytab`.
```

Comment:
Are we sure it allows symlinks? What about SELinux? Did we solve that?

Reply:
The documentation *should* have been updated for this feature. Lets link to the updated documentation here.

Reply:
The test case in the [commit](https://github.com/cockpit-project/cockpit/commit/9691029c99) only covers a file, not a symlink. @petervo, can you chime in here? Do symlinks work?

I added a link to the SSO documentation.

Reply:
I removed links from the documentation because they don't work all the time. Some do and some don't. What's in your example does work but ```/var/lib/ipa/gssproxy/http.keytab``` doesn't. So probably better to not refer to links

Reply:
Thanks, I dropped the sentence. @stefwalter , can you please re-review?